sdk/include/host/typedefs.h

```diff
@@ -27,6 +27,7 @@
 
 /* Basic definitions */
 #define UNIMPLEMENTED { printf("%s unimplemented\n", __FUNCTION__); exit(1); }
+#define UNIMPLEMENTED_ONCE { printf("%s unimplemented\n", __FUNCTION__); exit(1); }
```

Comment:
hmm maybe use `#define UNIMPLEMENTED_ONCE UNIMPLEMENTED`? :thinking:

Reply:
both could do with a better implementation, `exit(1)` is rather rude.

Reply:
yes actually I don't know why there is this `exit(1)`, it just makes the application exiting... :(